components/shared/Tailwind_Btn/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Link from 'next/link'
+import { Navigation } from 'components/enum/navigation'
+
+interface ButtonProps {
```

Comment:
Get rid of `onClick` for now, as we're not using it. It's also not correct, as we wouldn't have both a link and an onclick function.